lib/foreman/cron.rb

```diff
@@ -0,0 +1,89 @@
+require 'rake'
+
+module Foreman
+  module Cron
+    TASKS_BY_CADENCE = {
```

Comment:
When reading the code I wondered what the difference is between the tasks in here and the ones in `extra_tasks` - as in: what makes them *extra*?

We could also have something like `SUPPORTED_CANDENCES = [:hourly, :daily, :weekly, :monthly]` here and all the tasks are registered using `Foreman::Cron.register(…)` in some initializer.

Not blocking, but *I*'d find this cleaner :)